src/platform/user/profile/vet360/reducers/index.js

```diff
@@ -228,10 +228,10 @@ export default function vet360(state = initialState, action) {
       return {
         ...state,
         addressValidation: {
-          ...state.addressValidation,
+          ...initialAddressValidationState,
```

Comment:
rather than building on the current addressValidation state we should reset it to wipe out any old cruft and then populate it with the data from the error action.